ansible_builder/cli.py

```diff
@@ -2,28 +2,44 @@
 import sys
 
 from .main import AnsibleBuilder
+from . import constants
 
 
 def prepare(args=sys.argv[1:]):
     parser = argparse.ArgumentParser(prog='ansible-builder')
     subparsers = parser.add_subparsers(help='The command to invoke.')
 
-    build_command_parser = subparsers.add_parser('create',
-                                                 help='Outputs a build context, including a Containerfile, populated with dependencies.')
+    create_command_parser = subparsers.add_parser('create',
+                                                  help='Outputs a build context, including a Containerfile, populated with dependencies.')
 
-    build_command_parser.add_argument('-f', '--file',
-                                      default='execution-environment.yml',
-                                      dest='filename',
-                                      help='The definiton of the execution environment.')
+    build_command_parser = subparsers.add_parser('build',
+                                                 help='Builds the container with the Containerfile that got created via "create" command.')
 
-    build_command_parser.add_argument('-b', '--base-image',
-                                      default='shanemcd/ansible-runner',
-                                      help='The parent image for the execution environment.')
+    build_command_parser.add_argument('-t', '--tag',
+                                      default=constants.default_tag,
+                                      help='The name for the container being built.')
 
-    build_command_parser.add_argument('-c', '--context',
-                                      default=None,
-                                      dest='build_context',
-                                      help='The directory to use for the build context. Defaults to $PWD/context.')
+
+
+    for p in [create_command_parser, build_command_parser]:
+
+        p.add_argument('-f', '--file',
+                       default=constants.default_file,
+                       dest='filename',
+                       help='The definiton of the execution environment.')
+
+        p.add_argument('-b', '--base-image',
+                       default=constants.default_base_image,
+                       help='The parent image for the execution environment.')
+
+        p.add_argument('-c', '--context',
+                       default=None,
```

Comment:
looks like the test failures are due to this lacking a default, which you could put in `constants.default_build_context` for